### PR TITLE
Fix various typos recently introduced in the Transaction guide

### PR DESCRIPTION
**Pull request by gsmet**
See #4806 .

### CODE REVIEW

File: docs/src/main/asciidoc/transaction.adoc [hunk 1]
```diff
@@ -188,14 +188,14 @@ The default value is 60 seconds.
 
 == Configuring transaction node name identifier
 
-Narayana, as the underlaying transaction manager, has an concept of unique node identifier.
-This is important if you consider to use XA transactions with involve multiple resources.
+Narayana, as the underlying transaction manager, has a concept of unique node identifier.
```
Comment (geoand) [inline docs/src/main/asciidoc/transaction.adoc]: It probably should be `has a concept of a unique` (added `a`) if we are going to be really strict. I'll leave it up to you to decide

File: docs/src/main/asciidoc/transaction.adoc [hunk 2]
```diff
@@ -188,14 +188,14 @@ The default value is 60 seconds.
...
+This is important if you consider using XA transactions that involve multiple resources.
 
 The node name identifier plays a crucial part in the identification of a transaction.
 The node name identifier is forged into the transaction id when the transaction is created.
-Based on the node name identifier the transaction manager is capable to recognized the XA tranaction
+Based on the node name identifier, the transaction manager is capable to recognize the XA transaction
```
Comment (geoand) [inline docs/src/main/asciidoc/transaction.adoc]: ` to recognize` -> `of recognizing`

File: docs/src/main/asciidoc/transaction.adoc [hunk 3]
```diff
@@ -188,14 +188,14 @@ The default value is 60 seconds.
...
 counterparts created in database or JMS broker. The identifier makes possible for the transaction manager
-to roll-back the transaction counterparts during recovery.
+to roll back the transaction counterparts during recovery.
```
Comment (geoand) [inline docs/src/main/asciidoc/transaction.adoc]: +1

**Comment by gsmet** [comment]
I addressed all your comments.

**Review by geoand — APPROVED** [review]
Looks good!

**Pull request merged**